moped-editor/src/views/projects/projectView/ProjectComponents/EditComponentDrawTools.js

```diff
@@ -63,7 +63,7 @@ const EditComponentDrawTools = ({
   };
 
   const onUpdate = ({ features: updatedFeaturesArray, action }) => {
-    const wasComponentDragged = action === "move";
+    const wasComponentDragged = action === "move" || action === "change_coordinates";
     if (!wasComponentDragged) return;
 
     // Features not yet saved have a string id generated by Mapbox GL Draw with hat
```

Comment:
what is draw with hat?

Reply:
🎩 🤔 😄  Thanks for noticing this, Chia! @johnclary - could you add a link here to https://www.npmjs.com/package/hat for me since I could barely remember what my old comment meant? 🙏

Reply:
yeo—thanks! 🤠